pkgs/build-support/build-fhs-userenv-bubblewrap/env.nix

```diff
@@ -23,7 +23,9 @@
 
 let
   is64Bit = stdenv.hostPlatform.parsed.cpu.bits == 64;
```

Comment:
```suggestion
```
now unused

Reply:
still used https://github.com/NixOS/nixpkgs/blob/03c89f438e22cf0d0d22b882c2fe00faee18b72b/pkgs/build-support/build-fhs-userenv-bubblewrap/env.nix#L111

I tried doing `inherit (stdenv.hostPlatform) is64Bit`, however, that seems to be broken on aarch64-linux

Reply:
apparently this is because it should be `stdenv.is64bit`

Reply:
https://github.com/NixOS/nixpkgs/blob/master/lib/systems/inspect.nix